backend/src/main/java/ch/puzzle/okr/multitenancy/FlywayMultitenantMigrationInitializer.java

```diff
@@ -28,6 +29,7 @@ public void migrateFlyway() {
                     .configure() //
                     .dataSource(dataSourceConfig.url(), dataSourceConfig.name(), dataSourceConfig.password()) //
                     .locations(scriptLocations) //
+                    .placeholders(Map.of("tenant", tenantConfig.tenantId()))
```

Comment:
[med​-cg]: this is against our coding guidelines, use `//` as in the other lines or remove it everywhere but be consistent.